wahanegi/vibereport
# CI-551: [Snyk] Fix for 2 vulnerabilities

[![CI-551](https://badgen.net/badge/JIRA/CI-551/0052CC)](https://cloverpop-internship.atlassian.net/browse/CI-551) [<img width="16" alt="Powered by Pull Request Badge" src="https://user-images.githubusercontent.com/1393946/111216524-d2bb8e00-85d4-11eb-821b-ed4c00989c02.png">](https://pullrequestbadge.com/?utm_medium=github&utm_source=wahanegi&utm_campaign=badge_info)

<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to fix one or more vulnerable packages in the `rubygems` dependencies of this project.</h3>



#### Changes included in this PR

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - Gemfile


<details>
<summary>⚠️ <b>Warning</b></summary>

```
Failed to update the Gemfile.lock, please update manually before merging.
```
</details>


#### Vulnerabilities that will be fixed
##### With an upgrade:
Severity                   | Priority Score (*)                   | Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|-------------------------|:-------------------------|:-------------------------|:-------------------------
![medium severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/m.png "medium severity")  |  **429/1000**  <br/> **Why?** Has a fix available, CVSS 4.3  | Information Exposure <br/>[SNYK-RUBY-ACTIVESUPPORT-5851458](https://snyk.io/vuln/SNYK-RUBY-ACTIVESUPPORT-5851458) |  No  | No Known Exploit 
![medium severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/m.png "medium severity")  |  **616/1000**  <br/> **Why?** Proof of Concept exploit, Has a fix available, CVSS 5.9  | Web Cache Poisoning <br/>[SNYK-RUBY-RACK-1061917](https://snyk.io/vuln/SNYK-RUBY-RACK-1061917) |  Yes  | Proof of Concept 

(*) Note that the real score may have changed since the PR was raised.











Check the changes in this PR to ensure they won't cause issues with your project.



------------



**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs.*

For more information:  <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiI0YWMyMzcwYi0zNDI5LTQ1MTYtOWFiYy0xMjBlZGUwMzA5YTEiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6IjRhYzIzNzBiLTM0MjktNDUxNi05YWJjLTEyMGVkZTAzMDlhMSJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.snyk.io/org/rogergraves/project/6998e670-d362-4fcf-a4ad-ddff8c06149d?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)

🛠 [Adjust project settings](https://app.snyk.io/org/rogergraves/project/6998e670-d362-4fcf-a4ad-ddff8c06149d?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)

📚 [Read more about Snyk's upgrade and patch logic](https://support.snyk.io/hc/en-us/articles/360003891078-Snyk-patches-to-fix-vulnerabilities)

[//]: # (snyk:metadata:{"prId":"4ac2370b-3429-4516-9abc-120ede0309a1","prPublicId":"4ac2370b-3429-4516-9abc-120ede0309a1","dependencies":[{"name":"activeadmin","from":"2.13.1","to":"2.14.0"},{"name":"cssbundling-rails","from":"1.1.2","to":"1.2.0"},{"name":"devise-passwordless","from":"0.7.1","to":"1.0.0"},{"name":"jsbundling-rails","from":"1.1.1","to":"1.1.2"},{"name":"rails","from":"7.0.4.3","to":"7.1.0"},{"name":"rspec-rails","from":"6.0.2","to":"6.0.3"},{"name":"web-console","from":"4.2.0","to":"4.2.1"}],"packageManager":"rubygems","projectPublicId":"6998e670-d362-4fcf-a4ad-ddff8c06149d","projectUrl":"https://app.snyk.io/org/rogergraves/project/6998e670-d362-4fcf-a4ad-ddff8c06149d?utm_source=github&utm_medium=referral&page=fix-pr","type":"auto","patch":[],"vulns":["SNYK-RUBY-ACTIVESUPPORT-5851458","SNYK-RUBY-RACK-1061917"],"upgrade":["SNYK-RUBY-ACTIVESUPPORT-5851458","SNYK-RUBY-RACK-1061917"],"isBreakingChange":true,"env":"prod","prType":"fix","templateVariants":["pr-warning-shown","priorityScore"],"priorityScoreList":[429,616],"remediationStrategy":"vuln"})

---
## How to test

ActiveAdmin Functionality:

Go to the ActiveAdmin dashboard.
Navigate through all sections and ensure they function as they used to before the updates.
Pay close attention to search filters and sorting functionalities that use the ransack gem.

Ensure that search results and filters are as expected and no errors are displayed.
---

**Learn how to fix vulnerabilities with free interactive lessons:**

 🦉 [Learn about vulnerability in an interactive lesson of Snyk Learn.](https://learn.snyk.io/?loc&#x3D;fix-pr)

[CI-551]: https://cloverpop-internship.atlassian.net/browse/CI-551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
